modules/fonts/default.nix

```
@@ -0,0 +1,55 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let cfg = config.pinpox.defaults.fonts;
```

```suggestion
let cfg = config.awallau.defaults.fonts;
```
Otherwise `mkIf cfg.enable` won't work.